zuul.d/edpm_build_images.yaml

```diff
@@ -26,3 +26,14 @@
       - ^ci/playbooks/edpm_build_images/.*
       - ^scenarios/centos-9/edpm_build_images_ci.yml
       - ^zuul.d/edpm_build_images.yaml
+
```

Comment:
Move this from here to https://github.com/openstack-k8s-operators/ci-framework/blob/main/zuul.d/edpm_periodic.yaml so that we can find all periodic jobs easily

Reply:
done.